.github/actions/setup-rocm/action.yml

```diff
@@ -54,4 +54,8 @@ runs:
     - name: ROCm set GPU_FLAG
       shell: bash
       run: |
-        echo "GPU_FLAG=--device=/dev/mem --device=/dev/kfd --device=/dev/dri --group-add video --group-add daemon" >> "${GITHUB_ENV}"
+        if [[ ${{ runner.name }} == *-2 ]]; then
```

Comment:
Can you add a comment here explaining what's happening?

Reply:
I meant in the code haha, I understood the concept from your PR body, but a code reader may be confused.